client/src/components/Schedule/TableView.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  QuestionCircleOutlined,
+  YoutubeOutlined,
+  ChromeOutlined,
+  GithubOutlined,
+  SettingOutlined,
+} from '@ant-design/icons';
+import { Table, Tag, Tooltip } from 'antd';
+import { GithubUserLink } from 'components';
+import { dateSorter, getColumnSearchProps, tagsRenderer } from 'components/Table';
+import { CourseEvent } from 'services/course';
+import moment from 'moment-timezone';
+import { EventTypeColor, EventTypeToName, TaskTypes } from './model';
+
+type Props = {
+  data: CourseEvent[];
+  timeZone: string;
+};
+
+export function TableView({ data, timeZone }: Props) {
+  return (
+    <Table
+      rowKey={record => (record.event.type === TaskTypes.deadline ? `${record.id}d` : record.id).toString()}
+      pagination={false}
+      dataSource={data}
+      // onRow={() => ({
+      //   onClick: ({ target }) => {
+      //     if (!target.closest('a')) {
+      //       // eslint-disable-next-line no-console
+      //       console.log('row is clicked');
+      //     }
+      //   },
+      // })}
+      columns={[
+        {
+          title: columnFilter(),
```

Comment:
it will look more readable if we left `<SettingOutlined />` here. The function that contains `filter` in the name and defines the title is a bit confusing...

Reply:
fix